src/documents/classifier.py

```diff
@@ -174,7 +176,7 @@ def train(self):
 
         # Step 1: Extract and preprocess training data from the database.
         logger.debug("Gathering data from database...")
-        hasher = sha256()
+        hasher: _Hash = sha256()
```

Comment:
Just for something basic like this, I don't see a lot of value in typing with a protected item